src/components/Header/index.module.scss

```diff
@@ -15,3 +15,15 @@
     min-width: 0;
   }
 }
+
+.StickyContainer {
```

Comment:
The name style is a bit weird, but I found it's used in many places in this project. Maybe we should follow 1 well-known style since now.

Reply:
I also think that the current naming conventions are a bit confusing, but we'll deal with that in another issue.
I just created it: https://github.com/Magickbase/ckb-explorer-public-issues/issues/442